tools/src/keygen/Main.hs

```diff
@@ -146,6 +152,37 @@ getAvvmGenesis AvvmStakeOptions {..} = do
 rearrange :: (MonadIO m, MonadFail m, WithLogger m) => FilePath -> m ()
 rearrange msk = mapM_ rearrangeKeyfile =<< liftIO (glob msk)
 
+genPrimaryKey :: (MonadIO m, MonadFail m, WithLogger m) => FilePath -> m ()
+genPrimaryKey path = generateKeyfile True Nothing path >> pure ()
```

Comment:
`void $ generateKeyfile ...`
And yes, some `logInfo "successfully generated X"` would be good.